README.md

```diff
@@ -77,7 +82,11 @@ Belle is friends with Mrs. Potts
 Write your code here...
 
 ```ruby
-# code here
+
+friends = ["Chip Potts", "Cogsworth", "Lumière", "Mrs. Potts"]
+friends.each do |friend|
+  puts "Belle is friends with" friend
```

Comment:
You need a `+` between `"Belle is friends with"` and `friend`. Otherwise it errors out. Remember to test your code!

Otherwise, the `.each` statement looks good.